blog/_posts/2023-08-02-scala-week.md

```diff
@@ -0,0 +1,40 @@
+
+### Scala Spree
+
+The [**Scala Spree**](https://github.com/scalacenter/sprees) is scheduled for the 15th of September at 1:30 PM in Room 101 of the University of Madrid. The Spree brings together the Scala community to hack on open-source projects of local authors. If you're a local OSS maintainer or a contributor eager to participate, this event is a great opportunity to interact and collaborate. [**Register here**](https://airtable.com/appguX08g5OTDTZ4z/shrwSI11zJHmh7CkZ).
+
+### Tooling Summit & the SIP Meeting
+
+Besides the events open to the whole community, we will be conducting in-person meetings between people involved in various Scala-related groups and committees.
+We are will be organising the second iteration of the **Tooling Summit**. The Tooling Summit is a bi-annual gathering of the Scala tooling stakeholders where we ensure all tooling teams are in sync with each other and the compiler team. We will be following up on the progress done since the [last Tooling Summit](https://www.scala-lang.org/blog/2023/04/11/march-2023-scala-tooling-summit.html) and discussing the steps to take for the future development of the tooling ecosystem.
+Another event we are organising is the [**Scala Improvement Process (SIP)**](https://docs.scala-lang.org/sips/index.html). This process is crucial for deciding how the Scala language evolves, taking into account the opinions of all stakeholders.
+In-person summits of such kind are indispensable for better cohesion between all the stakeholders, as online platforms can only do so much in terms of forging connections.
+
+We look forward to a productive and engaging week of Scala at the University of Madrid! Big thanks to the local Scala Madrid user group for their support in organising this event. Stay tuned for more information and updates on these events.
+
+Here is a brief summary of all the events:
+
+| Event          | When        | Where                          | Accessible to public | Registration Link                                                                                   |
+|----------------|-------------|--------------------------------|----------------------|----------------------------------------------------------------------------------------------------|
+| SIP            | 11th Sept   | 17:30                          | ❌                   | N/A                                                                                                |
+| Scala Bridge   | 12th Sept   | 9:00 - 15:00, Room TBD         | ✅                   | [Register here](https://xebiaform.typeform.com/to/ZIWcng6z)                                       |
+| Tooling Summit | 12th Sept   | 8:30, Room TBD                 | ❌✅                 | N/A                                                                                                |
```

Comment:
What does the ❌✅ mean? The above section about the Tooling Summit does not explain who can access this event and how.